pipeline/bactmap.config

```diff
@@ -10,3 +10,11 @@
 
 profiles {
 }
+
+// using singularity platform
+singularity {
+  enabled    = true
+  autoMounts = true
+  cacheDir   = ''
+  pullTimeout = '*h'
+}
```

Comment:
While I typically think you should use singularity/apptainer, having this in the pipeline configuration seems odd and likely to break stuff?

I don't think I understand the intention here.